[2021-07-08 11:48:02,128] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T11:48:00.145019+00:00 [queued]>
[2021-07-08 11:48:02,156] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T11:48:00.145019+00:00 [queued]>
[2021-07-08 11:48:02,157] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 11:48:02,157] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 11:48:02,157] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 11:48:02,171] {taskinstance.py:1107} INFO - Executing <Task(PostgresOperator): query_users_data> on 2021-07-08T11:48:00.145019+00:00
[2021-07-08 11:48:02,179] {standard_task_runner.py:52} INFO - Started process 12157 to run task
[2021-07-08 11:48:02,186] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'query_users_data', '2021-07-08T11:48:00.145019+00:00', '--job-id', '58', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpaep_jof_', '--error-file', '/tmp/tmpb6kka16v']
[2021-07-08 11:48:02,191] {standard_task_runner.py:77} INFO - Job 58: Subtask query_users_data
[2021-07-08 11:48:02,256] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.query_users_data 2021-07-08T11:48:00.145019+00:00 [running]> on host 3cf0296804bd
[2021-07-08 11:48:02,323] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=query_users_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T11:48:00.145019+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T11:48:00.145019+00:00
[2021-07-08 11:48:02,325] {postgres.py:68} INFO - Executing: 
            SELECT * FROM users;
        
[2021-07-08 11:48:02,336] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 11:48:03,529] {dbapi.py:204} INFO - Running statement: 
            SELECT * FROM users;
        , parameters: None
[2021-07-08 11:48:03,925] {dbapi.py:212} INFO - Rows affected: 2
[2021-07-08 11:48:04,150] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=query_users_data, execution_date=20210708T114800, start_date=20210708T114802, end_date=20210708T114804
[2021-07-08 11:48:04,204] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 11:48:04,219] {local_task_job.py:151} INFO - Task exited with return code 0
